pkg/component/observability/monitoring/prometheus/shoot/scrapeconfigs.go

```diff
@@ -40,6 +45,7 @@ func CentralScrapeConfigs(namespace, clusterCASecretName string, isWorkerless bo
 						`{job="cadvisor",namespace="` + namespace + `"}`,
 						`{job="kube-state-metrics",namespace="` + namespace + `"}`,
 						`{__name__=~"metering:.+",namespace="` + namespace + `"}`,
+						`{job="` + druid + `",etcd_namespace="` + namespace + `"}`,
```

Comment:
For being consistent with the lines above and in the file referenced below I don't think druid as a const is beneficial
https://github.com/gardener/gardener/blob/799856cb1f6edfc56186fa608b7941e8b3b377d1/pkg/component/observability/monitoring/prometheus/aggregate/scrapeconfigs.go#L34